README.md

```diff
@@ -2,116 +2,82 @@
 A holistic approach to Backbone applications
 
 ## Build Commands:
```

Comment:
Let's chat about how we want to set up our readme. I think this is too much about the build.